src/app/core/auth/auth.service.ts

```diff
@@ -23,4 +25,16 @@ export class AuthenticationService {
   signUp(name: string, email: string, password: string){
     return from(createUserWithEmailAndPassword(this.auth, email, password))
   }
+
+  updateProfileData(profileData: Partial<UserInfo>): Observable<any> {
+    const user = this.auth.currentUser;
+     return of(user).pipe(
+       concatMap((user) => {
+         if (!user) throw new Error('Not authenticated');
+
+         return updateProfile(user, profileData);
+       })
+     );
```

Comment:
This method requires partial information of a user based off of the profileData interface to update the database.